ci/get_hdf5_win.py

```diff
@@ -114,8 +123,9 @@ def build_hdf5(version, hdf5_file, install_path, cmake_generator, use_prefix):
         copy(f, pjoin(install_path, 'lib'))
 
 
-def get_cmake_config_path(version, extract_point):
-    return pjoin(extract_point, REL_PATH_TO_CMAKE_CFG.format(version=version))
+def get_cmake_config_path(extract_point, zip_file):
+    dir_suffix = basename(zip_file).replace(".zip", "")
```

Comment:
Marginally frustrating that this requires Python 3.9 and I *just* proposed to drop 3.8
```suggestion
    dir_suffix = basename(zip_file).removesuffix(".zip")
```

Reply:
(please do not apply this)